engine/builtin.go

```diff
@@ -1268,7 +1269,7 @@ func Open(vm *VM, sourceSink, mode, stream, options Term, k Cont, env *Env) *Pro
 	case err == nil:
 		if s.mode == ioModeRead {
 			s.source = f
-			s.initRead()
+			_ = s.initRead()
```

Comment:
I’ve kept the code functionally the same since it’s a fork, only making cosmetic adjustments to satisfy the linter. I agree that these issues need attention, but this isn’t the right place to address them. It would be better to focus on fixing them in the upstream project.